projects/jupyter-server-ydoc/jupyter_server_ydoc/pytest_plugin.py

```diff
@@ -271,13 +271,16 @@ def _inner(
         last_modified: datetime | None = None,
         save_delay: float | None = None,
         store: SQLiteYStore | None = None,
+        writable: bool = False,
```

Comment:
Shouldn't it be writable by default?
```suggestion
        writable: bool = True,
```

Reply:
I think so. However, `FakeContentsManager` had `writable` set to `False` by default. So, just wanted to preserve it as I though it might be for some reasons.
If `"Writable": False` isn't necessary, do you want me to remove last commit and instead modify `Writable` to be `True` by default?

https://github.com/jupyterlab/jupyter-collaboration/blob/bb0e1fb8bb0c0e51f7118af640d84d197ed1b620/projects/jupyter-server-ydoc/jupyter_server_ydoc/test_utils.py#L24-L36

Reply:
Thanks, let's keep it like that then, and have it backwards-compatible.